css/style.css

```diff
@@ -1,19 +1,164 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
+body  {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    border: 2px solid red;
 }
 
+
 /**
  * Add your custom styles below
  * 
  * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
+ * - Be organized, use comments and separate your styles into meaningful chunks
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
 
+ 
+body{
+    width:100%;
+    display: flex;
```

Comment:
Why is the flex used here? Try removing the flex and margin properties. Are there any changes? If not, please remove them